packages/fiori/src/Wizard.js

```diff
@@ -747,7 +760,7 @@ class Wizard extends UI5Element {
 	}
 
 	get enabledStepsInHeaderDOM() {
```

Comment:
Since this getter now returns all steps, it should be renamed:

```suggestion
	get stepsInHeaderDOM() {
```

Reply:
This would be incompatible change in terms of API

Reply:
Then this one should be left as it previously was and a new one can be created.

Reply:
Why?